docs/root/api-v1/cluster_manager/cluster_hc.rst

```diff
@@ -80,3 +81,11 @@ service_name
   *(optional, string)* An optional service name parameter which is used to validate the identity of
   the health checked cluster. See the :ref:`architecture overview
   <arch_overview_health_checking_identity>` for more information.
+
+.. _config_cluster_manager_cluster_hc_redis_key:
+
+redis_key
```

Comment:
i considered overloading `path` in v1 instead of creating a new field. not sure if anyone else has a thought on that.

Reply:
I think what you've done looks good; the v1 APIs have a bounded lifetime in any case, so keeping them as straightforward as possible makes sense.